enaml/qt/docking/xbms.py

```diff
@@ -41,21 +42,21 @@ def __init__(self, width, height, data):
 
         """
         assert len(data) == (width * height)
-        bytes = []
-        for row in xrange(height):
+        bytes_list = []
```

Comment:
`bytearray` seems the more suitable type here

Reply:
I think that Bytes does not accept bytearray (we may consider changing this in atom, but it is outside the scope of this PR).